plugin-code/src/test/java/com/trivago/cluecumber/rendering/pages/pojos/pagecollections/PojoTest.java

```diff
@@ -14,7 +14,7 @@
 import java.util.List;
 
 public class PojoTest {
-    private static final int EXPECTED_CLASS_COUNT = 11;
+    private static final int EXPECTED_CLASS_COUNT = 12;
```

Comment:
I checked out your fork and this test expects 11, what's the result for you?

Reply:
My bad. I remember updating this when I added a new class but must have forgotten to revert it later on. Fixed now.